gcp/api/server.py

```diff
@@ -1179,6 +1225,47 @@ def is_cloud_run() -> bool:
   return os.getenv('K_SERVICE') is not None
 
 
+def get_gcp_project():
+  """Get the gcp project name."""
+  # We don't set the GOOGLE_CLOUD_PROJECT env var explicitly, and I can't find
+  # any confirmation on whether Cloud Run will set automatically.
+  # Grab the project name from the (undocumented?) field on ndb.Client().
+  # Most correct way to do this would be to use the instance metadata server
+  # https://cloud.google.com/run/docs/container-contract#metadata-server
+  return getattr(_ndb_client, 'project', 'oss-vdb')  # fall back to oss-vdb
+
+
+def _is_affected(ecosystem: str, version: str,
+                 affected_package: osv.AffectedPackage) -> bool:
+  """Checks if a version is affected."""
+  affected = False
+  ecosystem_info = ecosystems.get(ecosystem)
+  queried_version = ecosystem_info.sort_key(version)
+
+  for r in affected_package.ranges:
+    r: osv.AffectedRange2
+
+    for event in osv.sorted_events(ecosystem, r.type, r.events):
```

Comment:
nit, all  3 of these branches seem mutually exclusive, can we just add continue after each one? (or use elseif)